etc/korrel8r/rules/otellog.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+  - name: OtelLogToPod
+    start:
+      domain: otellog
+    goal:
+      domain: k8s
+      classes: [Pod]
+    result:
+      query: |-
+        k8s:Pod:{namespace: "{{.kubernetes.namespace_name}}", name: "{{.kubernetes.pod_name}}"}
```

Comment:
The OTEL field is "k8s.namespace.name", see https://opentelemetry.io/docs/specs/semconv/resource/k8s
Why the Viaq name here?